tmt-app/src/Components/ResultsCards.jsx

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import { Stack } from "@mui/material";
+import CustomCard from "./CustomCard";
+import { MOCK_DATA } from "../constants/mockData";
+
+function ResultsCards() {
+  return (
+    <Stack spacing={3}>
+      <CustomCard dummyData={MOCK_DATA} />
```

Comment:
Ah looking at this component name, I would expect one element to render, but recalling the code that's in customCard, we're passing in an array of item ms we iterate over. That logic should be here, or we should reconsider some components' names